priv/templates/phx.gen.auth/confirmation_instructions_live.ex

```diff
@@ -8,7 +8,7 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
     <.header>Resend confirmation instructions</.header>
 
     <.simple_form :let={f} for={:<%= schema.singular %>} id="resend_confirmation_form" phx-submit="send_instructions">
-      <.input field={{f, :email}} type="email" label="Email" required />
+      <.input field={{f, :email}} type="email" label="Email" autocomplete="username" required />
```

Comment:
Is there a reason why you are using `autocomplete="username"` rather than `autocomplete="email"` on the email fields?

Reply:
That couldn't hurt. I'm not a maintainer myself, but I'd be keen to see it merged.

Reply:
> Not really, I think you're right and we should update this to `autocomplete="email"`. I'm happy to do so and also bring this PR in sync with the master branch?

autocomplete="username" with type=“email” is correct as those are the attributes of the same field in other templates (e.g. registration). So with the current value the browser is advised to use the same value the user as entered before.

HTML spec allows the combination of type=“email” and autocomplete=username”. 
https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#control-group-username

Chromium advices the combi:
https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands/

Reply:
Thanks @BartOtten. I'll bring this PR up to date with master later this week.